.github/CONTRIBUTING.md

```diff
@@ -30,6 +30,65 @@ SimpleITK uses a branchy workflow where each PR should be a new topic branch. Lo
 
 After completing the local branch, push it to your fork, then create a GitHub pull request in the SimpleITK upstream repository. The pull request is then required to have an approving *code review*, and the *continuous integration* testing pass before the topic is merged into the upstream repository.
 
+### How to contribute with the right workflow using github commands
+
+``` shell
+# Fork the repo
+
+# clone to your laptop
+git clone https://github.com/<username>/SimpleITK.git
+
+# go to the folder
+cd SimpleITK
+
+# check the repo
+git remote -v
+origin  https://github.com/<username>/SimpleITK.git (fetch)
+origin  https://github.com/<username>/SimpleITK.git (push)
+
+# add the upstream
+git remote add upstream https://github.com/SimpleITK/SimpleITK.git
+
```

Comment:
Please break this into two parts. The part above this comment is only performed once during initial setup. The part below this comment should be performed every time you want to contribute.